cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -193,6 +194,12 @@ def render_files(reldata=None):
     target = os.path.join(rendered_manifest_dir, 'kubedns-rc.yaml')
     # Render files/kubernetes/kubedns-rc.yaml for the DNS pod.
     render('kubedns-rc.yaml', target, context)
+    # when files change on disk, we need to inform systemd of the changes
+    try:
+        check_call(['systemctl', 'daemon-reload'])
```

Comment:
Why wouldn't we recycle/reload just the 3 kubernetes services? The way I read the daemon-reload it reloads the systemd manager configuration and reload all unit files.